oeps/processes/oep-0001.rst

```diff
@@ -437,20 +437,15 @@ to capture the decision(s), as deemed appropriate by the Authors and Arbiter.
 To help guide Authors, here are a few ready-made templates that are available
 for use:
 
-* :ref:`PEP-based template <pep_based_template>` based on Python's PEP_ standard.
-* :ref:`ADR-based template <adr_based_template>` based on `Architecture Decision Records`_.
-* :ref:`External link template <external_link_template>` for OEPs with mostly external content.
+* `PEP-based template`_ based on Python's PEP_ standard.
+* `ADR-based template`_ based on `Architecture Decision Records`_.
+* `External link template`_ for OEPs with mostly external content.
 
-..
-  Add a hidden toctree to indicate that these files are referenced from here in the overall
-  table of contents.
 
-.. toctree::
-    :hidden:
+.. _PEP-based template: https://github.com/openedx/open-edx-proposals/tree/master/oep-templates/pep-based-template.rst
+.. _ADR-based template: https://github.com/openedx/open-edx-proposals/tree/master/oep-templates/adr-based-template.rst
+.. _External link template: https://github.com/openedx/open-edx-proposals/tree/master/oep-templates/external-link-template.rst
```

Comment:
these links won't work until this merges

Reply:
Why make these external links?  You should refer to the new location directly so that we can detect broken links as a part of the build process in the future: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#links-between-rst-documents

Reply:
I thought the external links were a good solution, since they are meant to be used as .rst files, not as published HTML rendered from .rst files.  The links-between-rst-documents will render the templates, which isn't helpful.

Reply:
Ah, makes sense, I think I would have gone with the [literalinclude](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude) directive for this but I don't feel strongly enough about that to hold this up.